style.css

```diff
@@ -577,6 +561,70 @@ form {
   border-color: #2230d2;
 }
 
+/* Mobile Menu */
+#mobile-menu {
+  display: none; /* Initially hidden */
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(50, 70, 255, 0.8);
+  backdrop-filter: blur(10px); /* Apply a blur effect to the backdrop */
+  z-index: 9999; /* On top of other content */
+  padding: 86px 28px; /* Updated padding */
+  box-sizing: border-box;
+  overflow: auto;
+}
+
+#mobile-menu ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+#mobile-menu ul li {
+  margin-bottom: 50px; /* Use margin instead of gap */
+}
+
+#mobile-menu ul li a {
+  display: flex;
+  width: 251px;
+  height: 24px;
+  flex-direction: column;
+  justify-content: center;
+  flex-shrink: 0;
+  color: #fff; /* White text */
+  font-size: 32px;
+  font-family: var(--main-font);
+  font-weight: 600;
+  line-height: 44px;
+  text-decoration: none;
+  z-index: 1111111; /* Ensure menu items are on top */
+}
+
+/* Close Icon */
+#close-icon {
+  display: flex;
+  position: absolute; /* Position the icon absolutely */
+  top: 20px; /* Position from the top */
+  right: 20px; /* Position from the right */
+  width: 24px;
```

Comment:
- [OPTIONAL] Comments are best used selectively and strategically so that they do not make readability difficult, and also to avoid bugs. If you have gotten some assistance from AI technologies, I think removing the comments meant as an explanation would be a rewarding exercise for you to keep your code base clean.